### ISSUE TITLE
AMP Ads: "Advertisement" label should be displayed above ad

### ISSUE THREAD

**Issue by Bopchy** [bbc/simorgh#6449]
**Is your feature request related to a problem? Please describe.**
An "Advertisement" label should be displayed above ad as shown in the [designs](https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5e790b9088869d6a1a17279b).

The GEL group that this should be using is GEL Minion.
The colour is 'Rhino'.

For RTL langauges, the label should appear on the opposite side to the designs. We can confirm if the GNL solution for canonical does this as well.

**Describe the solution you'd like**
The `Advertisement` label should be localised/translated depending on the service.

Translations can be taken from gnl-ngas https://github.com/bbc/gnl-ngas **but need confirmation these are complete**.

https://github.com/bbc/gnl-ngas/blob/develop/src/ads/locale.js

Accessibility criteria:
Given there is advertising on the page
When the ‘ADVERTISEMENT’ label is displayed
Then this must meet the colour contrast ratio of 4.1:5 (as this is key to understanding)

We should perform an a11y **DEV** swarm for this issue.

**Describe alternatives you've considered**
N/A.

**Testing notes**
- Cover that the label is present at the unit test level.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

**Comment by sareh** [comment]
Please note that we need to consider these accessibility acceptance criteria when adding this Advertisement label: https://github.com/bbc/simorgh/issues/5956

**Comment by sareh** [comment]
Translations are currently missing for Serbian (sr-Lat, sr-Cyrl) and Yoruba (yo). We should be able to complete this issue with existing translations and for Serbian/Yoruba use the English text `Advertisement`. 
Then we can raise another issue to add the remaining missing translations once we have them.
